tests/framework/installer/ceph_settings.go

```diff
@@ -85,6 +85,7 @@ func (s *TestCephSettings) readManifestFromGitHubWithClusterNamespace(filename,
 }
 
 func (s *TestCephSettings) replaceOperatorSettings(manifest string) string {
+	manifest = strings.ReplaceAll(manifest, `ROOK_LOG_LEVEL: "INFO"`, `ROOK_LOG_LEVEL: "DEBUG"`)
```

Comment:
I'm a bit hesitant to always enable the debug logs in all the tests, but let's try it for now